src/WDCMethods.js

```diff
@@ -97,6 +97,16 @@ const sanitizeVariableName = variableName => {
   return variableName.replace(/\s/g, "_").replace(/[^a-zA-Z0-9_]/g, "");
 };
 
+/*
+check if data matches -999999 and return null instead
+*/
+const checkForNull = data => {
```

Comment:
I know this is a simple function, but could we add a unit-test for it?

Reply:
Good call. I added one.